examples/Dockerfile-envoy-image

```diff
@@ -4,3 +4,6 @@ ADD build/source/exe/envoy /usr/local/bin/envoy
 RUN apt-get update && apt-get install -y \
     curl \
     python-pip
+RUN objcopy --only-keep-debug /usr/local/bin/envoy /usr/local/bin/envoy.dbg
```

Comment:
Ok, dropping debug. The curl and python above are needed in the examples. I could move their addition to the examples' docker file. If we want that, then I would move this Dockerfile to ci/; at that point this dockerfile would belong there rather than here. What do you think?